addons/website/views/snippets/snippets.xml

```diff
@@ -681,7 +681,8 @@
 
     <!-- Grid mode columns -->
     <div data-js="GridColumns"
-         data-selector=".row > div">
+         data-selector=".row > div"
+         data-exclude=".s_website_form_field, :has(.s_website_form_input)">
```

Comment:
Could you check the other `.row > div` selectors where we added stuff like `.s_col_no_resize` in the `data-exclude`
It will therefore still not be possible to have these columns in the options panel and still not possible to delete them.

![image](https://github.com/odoo/odoo/assets/52911687/0a70f24a-926b-4346-a06d-34b683ff6a6e)